app/src/main/java/com/x8bit/bitwarden/data/autofill/parser/AutofillParserImpl.kt

```diff
@@ -77,7 +77,9 @@ class AutofillParserImpl(
             .flatten()
 
         // Find the focused view.
-        val focusedView = autofillViews.firstOrNull { it.data.isFocused }
+        val focusedView = autofillViews
+            .firstOrNull { it.data.isFocused }
+            ?: autofillViews.firstOrNull { it !is AutofillView.Unused }
```

Comment:
I feel like this warrants some kind of comment explaining why we'd fallback to a view that is not focused.